static/app/views/performance/browser/resources/resourceSummaryPage/sampleImages.tsx

```diff
@@ -43,27 +46,40 @@ function SampleImages({groupId}: Props) {
     })
     .splice(0, 5);
 
+  const hasImages = filteredResources.length > 0;
+
+  let body: React.ReactNode;
+
+  if (!showImages) {
+    body = <DisabledImages onClickShowLinks={() => setShowImages(true)} />;
+  } else if (showImages && isLoadingImagesLoading) {
+    body = <LoadingIndicator />;
+  } else if (showImages && !hasImages) {
+    body = (
+      <EmptyStateWarning>
+        <p>{t('No results found for your query')}</p>
```

Comment:
```suggestion
        <p>{t('No images detected')}</p>
```

I wonder if we should capture a Sentry error when we hit this so that we gather some examples of when someone goes to an images resource summary and sees no images. Do you think that is a good idea?

Reply:
I def l ike the idea, it does not entirely fall into an error category I would say, as we do expect missing images when there simply isn't enough volume and we sample them out. But we will get a bunch of data on this to help us optimize sampling if needed. Is this a better case for DDM, or would we not get any context like the customer itself.

Reply:
We can't use DDM because of cardinality with trying to attach org, errors you can at least see org slug tagged hence why I suggested that.

Reply:
Makes sense!